docs/configuration/README.md

```diff
@@ -388,7 +388,7 @@ kvstore:
       - <string>
 
     # The Dial timeout for the ETCD connection.
-    [dial_tmeout: <duration> | default = 10s]
+    [dial_timeout: <duration> | default = 10s]
```

Comment:
are we 100% sure this is a typo in the docs and not a config actually spelled wrong?

Reply:
Afaict, the only instances of that are the document itself:
https://www.google.com/search?q=%22dial_tmeout%22